Dockerfile

```diff
@@ -91,6 +91,12 @@ RUN apt-get update -q && \
     phpdismod pdo_pgsql && \
     phpdismod pgsql && \
     phpdismod xdebug && \
+    # Remove extra extensions installed via packages for other versions of PHP
+    rm -rf /usr/lib/php/20121212 && \
+    rm -rf /usr/lib/php/20131226 && \
+    # rm -rf /usr/lib/php/20151012 && \
```

Comment:
that version is the active php engine folder. there are 4 known engines, I commented out the active one in each case